git_machete/cmd.py

```diff
@@ -626,7 +626,7 @@ def add(cli_ctxt: CommandLineContext, b: str) -> None:
         expect_in_managed_branches(onto)
 
     if b not in local_branches(cli_ctxt):
-        rb: Optional[str] = get_sole_remote_branch(cli_ctxt, b)
+        rb: Optional[str] = sole_remote_branch(cli_ctxt, b)
```

Comment:
sole_remote_branch name suggest that it is a variable containing branch, where in reality it is a method returning such. By removing 'get_' from the names of method it now looks like they these methods are variables. It's established programming convention to name methods as actions (like 'get_string_length' or 'notify_police') or questions (like 'is_even(number) or 'are_commits_in_the_same_format'). I don't recommend introducing this change and similar ones. Applies to all removal of 'get_' or 'is_' from methods and functions names.

Reply:
Side Note, python itself is rather into the idea of using 'is' methods names. See https://docs.python.org/3/library/stdtypes.html#float.is_integer or https://docs.python.org/3/library/functions.html#isinstance